data/presets/highway/crossing/uncontrolled.json

```diff
@@ -17,6 +17,5 @@
         "key": "crossing",
         "value": "uncontrolled"
     },
-    "name": "Marked Crosswalk",
-    "searchable": false
+    "name": "Marked Crosswalk"
```

Comment:
`uncontrolled` isn’t just “marked”, it’s “marked without traffic signals”.

Reply:
Right, but we also need to keep a balance between keeping the preset names short and including every detail of a preset in its name. I thought that the presence of the dedicated preset for crossings with traffic signals makes it clear enough that the "marked crosswalk" preset is not meant for situations where there is a traffic light. There's also the :information_source:  icon which shows this additional detail for the preset. Last but not least, the name _"Marked Crosswalk"_ worked well in the past for the tag [`crossing=marked`](https://wiki.openstreetmap.org/w/index.php?title=Tag:crossing%3Dmarked&oldid=2400462) which also implied the absence of traffic signals.

Maybe you have a better name for this preset (I'd rather avoid naming it _Uncontrolled Crosswalk_ :wink: )?

Reply:
I think “Marked Crosswalk” is passable as long as there’s a field that can transform it into `crossing=traffic_signals`. It looks like you’ve left the Type field in place for this preset, which helps somewhat. The field isn’t as usable to non-English speakers, but providing localizable option strings would be fraught: several of the most common values mean the same thing – or don’t mean the same thing at all, depending on who you ask – and each has its ardent defenders.